test/e2e/storage/volume_metrics.go

```diff
@@ -165,10 +155,26 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 			kubeletmetrics.VolumeStatsInodesUsedKey,
 		}
 
-		for _, key := range volumeStatKeys {
-			kubeletKeyName := fmt.Sprintf("%s_%s", kubeletmetrics.KubeletSubsystem, key)
-			verifyVolumeStatMetric(kubeletKeyName, pvc.Namespace, pvc.Name, kubeMetrics)
-		}
+		waitErr := wait.Poll(30*time.Second, 5*time.Minute, func() (bool, error) {
+			// Grab kubelet metrics from the node the pod was scheduled on
```

Comment:
Consider adding a `By` or a `Logf` here so when debugging failures we can know which step the test was on.

Reply:
👍 - done